tests/integration/mock/CodeActAgent/test_browse_internet/prompt_004.log

```diff
@@ -118,14 +118,13 @@ RootWebArea 'The Ultimate Answer', focused
 	[8] heading 'The Ultimate Answer'
 	[9] paragraph ''
 		StaticText 'Click the button to reveal the answer to life, the universe, and everything.'
-	[10] button 'Click me', focused
+	[10] button 'Click me', clickable, focused
 	StaticText 'The answer is OpenDevin is all you need!'
```

Comment:
That looks strange. my another PR also failed this test. But my PR doesn't related to this change.

Reply:
But if some PR trigger this change, why that PR can pass the integration check. I worry our test ignore some corner.

Reply:
because we don't enforce a CI run with latest main branch before merging.

Reply:
Ah. That might be it. Two PRs related to browsing agent were merged close one after the other.

Reply:
I guess one of those was mine? Apologies for that, I should've reverted one commit in time. 😢

Reply:
No, @tobitege, when you encountered an integration test failure, it was because two other PRs somehow conflicted to each other in a non-obvious way. Otherwise we can't merge in a way that makes integration tests fail.